frontend/src/pages/ArtifactList.test.tsx

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import { ArtifactList } from './ArtifactList';
+import { PageProps } from './Page';
+import { RoutePage } from '../components/Router';
+import TestUtils from '../TestUtils';
+import { render, screen } from '@testing-library/react';
+import {
+  Api,
+  Artifact,
+  ArtifactType,
+  GetArtifactsResponse,
+  GetArtifactTypesResponse,
+  Value,
+} from '@kubeflow/frontend';
```

Comment:
Note, we should soon move `@kubeflow/frontend` back into KFP repo, because KFP is the only consumer now. At that time, you'll need to update the import everywhere.

Reply:
Thank you for the heads-up! Noted that it will require import changes while merging frontend dependency to KFP.